sequencer/src/api.rs

```diff
@@ -5640,8 +5641,11 @@ mod test {
 
             // verify auth root if the consensus version is v4
             if header.version() == DrbAndHeaderUpgradeVersion::VERSION {
-                let auth_root = state_cert_v2.auth_root.expect("auth root is not None");
-                let header_auth_root = header.auth_root().unwrap().unwrap();
+                let auth_root = state_cert_v2.auth_root;
+                let header_auth_root = header.auth_root().unwrap();
+                if auth_root.is_zero() || header_auth_root.is_zero() {
+                    panic!("auth root shouldn't be zero");
```

Comment:
just checking, is it okay to panic here? or return err?

Reply:
this is for V4 header where we do calculate the auth_root field so yeah its fine to panic here if its not zero